src/bp/nlu/stan/model-repo.ts

```diff
@@ -27,8 +28,8 @@ interface DBDriver {
 export type ModelRepoOptions = FSDriver | DBDriver
 
 interface ModelOwnershipOptions {
-  appId?: string
-  appSecret?: string
+  appId: string
```

Comment:
sometimes appId and appSecret will be undefined am I right ? will it crash ?

Reply:
they'll never as Joi sets empty strings for undefined appid and secret. This is unit tested in validation.test.ts